### ISSUE TITLE
PEP 8 -- Style Guide for Python Code

### ISSUE THREAD

**Issue by o-smirnov** [ratt-ru/reading-list#4]
As suggested by @gijzelaerr, mandatory reading for all Python coders.

http://legacy.python.org/dev/peps/pep-0008/

**Comment by griffinfoster** [comment]
essential

**Comment by o-smirnov** [comment]
Indeed. I have even created a "mandatory" label especially for this. We're going to have a quick exam on PIP 8 at the next group meeting -- failing students to be immediately fired and deported from the country. :)

**Comment by IanHeywood** [comment]
If you guys have time to follow all these guidelines then you're not working hard enough.

**Comment by o-smirnov** [comment]
That's also true. Still, I'm hoping some of this seeps through.

**Comment by griffinfoster** [comment]
google is pretty good, they have released a very simple tool called yapf which will properly format your code to the PEP8 standard. it is easy to install with pip and run on your code.

https://github.com/google/yapf
